### PR TITLE
GrowthViz Docker

### PR DESCRIPTION
**Pull request by RobertScalfani**
This Pull Request includes the settings, Dockerfile, and README updates used to create and use the GrowthViz Docker image.

Notes:
- Not sure why there is such a large difference with GrowthViz.ipynb?
- Not sure if output/cleaned_obs.csv should be deleted or in a .gitignore or?

### CODE REVIEW

**Comment by eedrummer** [comment]
Getting Qgrid to work may be as simple as adding: `jupyter nbextension enable --py --sys-prefix qgrid` to the `Dockerfile`

**Comment by RobertScalfani** [comment]
Yup, the addition to the Dockerfile you suggested fixed Qgrid. Removed `__pycache__` but was not able to find the `.DS_Store` file on git or my local machine, even when unhiding hidden files. I must be missing something there.

**Comment by dchud** [comment]
In the `README`, in step 4, it references "the `/growthviz` folder you created in step 1". Should that be `growthviz-data`?

**Comment by RobertScalfani** [comment]
Yes @dchud , nice catch I will fix that now.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+RUN pip install pandas
+RUN pip install matplotlib
+RUN pip install ipywidgets
+RUN pip install seaborn
+RUN pip install qgrid
+
+COPY . ./
+
+RUN mkdir growthviz-data/output/
```
Comment (eedrummer) [inline Dockerfile:15]: I think this needs to be a `mkdir -p`. Trying to do a build on my machine is failing.

**Pull request merged**